### ISSUE TITLE
Windows Terminal integrated into Visual Studio cannot open powershell

### ISSUE THREAD

**Issue by kitattyor** [microsoft/terminal#8050]
# Environment

```none
Windows build number:  Microsoft Windows [Version 10.0.18363.1139]
Windows Terminal version (if applicable): 1.3.2651.0

Any other software?
Visual Studio: 16.7.5

```

# Steps to reproduce

I have been redirected here from Visual Studio support. 

I am getting a `[error 0x8007010b when launching powershell.exe]` error when trying to use the windows terminal integrated into VS. I have selected the standard “Windows Terminal” Settings in options menu.

Running wt /d "D:\Projects\<path-to-project>" from the developer command prompt works as expected. 

![2020-10-27 10_32_56-Server1main - Microsoft Visual Studio](https://user-images.githubusercontent.com/45325249/97250081-2143d800-1840-11eb-810c-793c01d6ecb7.png)

![2020-10-27 10_36_06-Server1main - Microsoft Visual Studio](https://user-images.githubusercontent.com/45325249/97250132-43d5f100-1840-11eb-87b6-2c6a3b578c9d.png)

The developer powershell works as expected. 

# Expected behavior

Should open up windows terminal properly. 

# Actual behavior

Refer to screenshots above.

**Comment by DHowett** [comment]
Is `%folder%` supposed to work? Where is it documented? This looks like the error message you might get if `%folder%` doesn't exist.

**Comment by kitattyor** [comment]
You are right. The `"%folder%"` seems to be the issue. It's the default argument for the Windows Terminal option in Visual Studio. 

Documented in the lower panel too: 
![2020-10-27 10_51_08-Server1main - Microsoft Visual Studio](https://user-images.githubusercontent.com/45325249/97251242-71239e80-1842-11eb-9c83-054c4872cae8.png)

It works if I manually specify the the file path.

**Comment by DHowett** [comment]
This is interesting! And you're not using the preview version of Terminal...
Thanks!

**Comment by zadjii-msft** [comment]
@javierdlg as an FYI

**Comment by javierdlg** [comment]
It's worth mentioning that when a shell is spawned via VS (via the terminal or using the developer Cmd/PowerShell), it also inherits the environment variables that VS sets. So `%folder%` is most likely a variable that VS sets and isn't available outside of the process.

A workaround for this could be to spawn the shell and import the VS module. You can do this _and_ use the windows terminal if you copy the arguments used to spawn a shell process under Tools->Options->Terminal->Arguments and use that to spawn your own terminal process.

tl;dr use `-NoExit -Command "& { Import-Module $env:VSAPPIDDIR\..\Tools\Microsoft.VisualStudio.DevShell.dll}; Enter-VsDevShell -SkipAutomaticLocation -SetDefaultWindowTitle -InstallPath $env:VSAPPIDDIR\..\..\` as your arguments and add any extra arguments.

**Comment by DHowett** [comment]
Is there a chance that this changed with Terminal 1.4?

Weird enough, I can't even _find_ that option in my VS. That's very strange.

**Comment by javierdlg** [comment]
Sorry I skipped one subfolder. It should be under `Tools->Options->Environment->Terminal->Arguments`

**Comment by kitattyor** [comment]
Sorry for the late reply. But no, it hasn't changed anything.

**Comment by kitattyor** [comment]
@javierdlg @DHowett when I add 

`-NoExit -Command "& { Import-Module $env:VSAPPIDDIR\..\Tools\Microsoft.VisualStudio.DevShell.dll}; Enter-VsDevShell -SkipAutomaticLocation -SetDefaultWindowTitle -InstallPath $env:VSAPPIDDIR\..\..\ `

I get an error message : The following arguments were not expected: -Command -NoExit
